### PR TITLE
Feat/create footer component

### PR DESCRIPTION
**Pull request by danielfpadilla**
#28 
Create footer component and use DaisyUI components

**Large screens**
image

**Small screens**
image

### CODE REVIEW

File: src/components/commons/Footer.vue [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    </a>
+    <a href="https://bit.ly/3xjDvN2"
+      target="_blank">
+      <button class="btn btn-ghost text-white">
+        <i class="la la-newspaper"></i>
+        Press Kit
+      </button>
+    </a>
+  </div>
+  <p>Copyright © 2023 Open Source Software PH. All Rights Reserved.</p>
```
Comment (jofftiquez) [inline src/components/commons/Footer.vue:33]: Make the year dynamic. You can use vue.js interpolation directly here. Like so:

```js
{{new Date().getFullYear()}}
```

File: src/components/commons/Footer.vue [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<template>
+<footer class="footer footer-center p-10 text-primary-content ossph-gradient-bg">
+  <div class="grid grid-flow-row sm:grid-flow-col gap-4">
+    <a href="https://discord.com/servers/open-source-software-ph-905496362982981723"
+      target="_blank">
+      <button class="btn btn-ghost text-white">
```
Comment (jofftiquez) [inline src/components/commons/Footer.vue:6]: Please set button width to full/block on mobile or small screens.

image

**Review by jofftiquez — CHANGES_REQUESTED** [review]

File: src/components/commons/Footer.vue (hunk 2 above)
Comment (danielfpadilla) [reply src/components/commons/Footer.vue:6]: cge boss. good idea

Comment (danielfpadilla) [reply src/components/commons/Footer.vue:6]: Applied this behavior

**Wide screen**
image

**Smaller screens**
image

File: src/components/commons/Footer.vue (hunk 1 above)
Comment (jofftiquez) [reply src/components/commons/Footer.vue]: Sir @danielfpadilla in case you missed this one, just notifying you. Thank you <3

Comment (danielfpadilla) [reply src/components/commons/Footer.vue]: Gosh I really missed this one lol

Comment (danielfpadilla) [reply src/components/commons/Footer.vue]: This is done

**Review by jofftiquez — APPROVED** [review]

**Pull request merged**